src/spatialdata/datasets.py

```diff
@@ -171,21 +196,27 @@ def _generate_blobs(self, length: int = 512, seed: Optional[int] = None) -> Arra
         mask = gaussian_filter(mask, sigma=0.25 * length * 0.1)
         return mask
 
-    def _points_blobs(self, length: int = 512, n_points: int = 200) -> DaskDataFrame:
+    def _points_blobs(
+        self, transforms: Optional[dict[str, Any]] = None, length: int = 512, n_points: int = 200
+    ) -> DaskDataFrame:
         rng = default_rng(42)
-        arr = rng.integers(10, length - 10, size=(n_points, 2)).astype(np.int_)
+        arr = rng.integers(10, length - 10, size=(n_points, 2)).astype(np.int64)
         # randomly assign some values from v to the points
-        points_assignment0 = rng.integers(0, 10, size=arr.shape[0]).astype(np.int_)
+        points_assignment0 = rng.integers(0, 10, size=arr.shape[0]).astype(np.int64)
         genes = rng.choice(["a", "b"], size=arr.shape[0])
         annotation = pd.DataFrame(
             {
                 "genes": genes,
                 "instance_id": points_assignment0,
             },
         )
-        return PointsModel.parse(arr, annotation=annotation, feature_key="genes", instance_key="instance_id")
+        return PointsModel.parse(
+            arr, transformations=transforms, annotation=annotation, feature_key="genes", instance_key="instance_id"
+        )
 
-    def _circles_blobs(self, length: int = 512, n_shapes: int = 5) -> GeoDataFrame:
+    def _circles_blobs(
+        self, transforms: Optional[dict[str, Any]] = None, length: int = 512, n_shapes: int = 5
```

Comment:
why not call it `transformations` instead of transforms, as it is also called int he models? If you change it here, please change it everywhere

Reply:
resolved with [bef3a35](https://github.com/scverse/spatialdata/pull/282/commits/bef3a353b97f515e5561d8672c88b55252b7a675)